docs/advanced.rst

```diff
@@ -581,7 +581,20 @@ For example:
     $ pipenv run echospam "indeed"
     I am really a very silly example indeed
 
-You can then display the names and commands of your shortcuts by running ``pipenv scripts`` in your terminal.
+You can also specify pacakge functions as callables such as: ``<pathed.module>:<func>``. These can also take arguments.
+For exaple:
+
+.. code-block:: toml
+
+    [scripts]
+    my_func_with_args = {call = "package.module:func('arg1', 'arg2')"}
+    my_func_no_args = {call = "package.module:func()"}
+
+::
```

Comment:
should this be:
```
.. code-block:: shell
```
?

Reply:
Maybe, but I was just following the same format as the rest of this file to match the other parts of this section (they don't use `..code-block:: shell` and just use `::`
If you want me to change it I can change it, should i change it for the whole section?